### PR TITLE
Feature/gen 13 db mongo connection

### PR DESCRIPTION
**Pull request by SaurabhChandel007**
mongodb added as db

### CODE REVIEW

File: admin-ui/prisma/schema.prisma [hunk 1]
```diff
@@ -116,34 +108,37 @@ model Project {
...
+  updatedAt      DateTime @updatedAt
+  name           String   
+  screenId       String?  @map("screen_id") @db.ObjectId
   screen         Screen? @relation(fields: [screenId], references: [id])
 
   @@map("tb_component")
 }
 
 
+model ProjectData{
```
Comment (vivekbarsagadey) [inline admin-ui/prisma/schema.prisma:133]: this need to rename this project json

File: admin-ui/prisma/schema.prisma [hunk 2]
```diff
@@ -94,9 +86,9 @@ model Company {
 
 
 model Project {
- id             String    @id @default(cuid())
- createdAt      DateTime  @default(now()) @map("created_at")
- updatedAt      DateTime  @updatedAt @map("updated_at")
+ id             String   @id @default(auto()) @map("_id") @db.ObjectId
+ createdAt      DateTime @default(now())
+ updatedAt      DateTime @updatedAt
```
Comment (vivekbarsagadey) [inline admin-ui/prisma/schema.prisma:91]: it should be default new

File: admin-ui/prisma/schema.prisma [hunk 3]
```diff
@@ -53,18 +53,10 @@ model User {
...
-   @@map("customers")     
-}
-
 model Company {
-  id             String    @id @default(cuid())
-  createdAt      DateTime  @default(now()) @map("created_at")
-  updatedAt      DateTime  @updatedAt @map("updated_at")
+  id             String   @id @default(auto()) @map("_id") @db.ObjectId
+  createdAt      DateTime @default(now())
+  updatedAt      DateTime @updatedAt
```
Comment (vivekbarsagadey) [inline admin-ui/prisma/schema.prisma:59]: check is there any default value

File: admin-ui/prisma/schema.prisma [hunk 4]
```diff
@@ -42,7 +42,7 @@ model Session {
 }
 
 model User {
-  id            String    @id @default(cuid())
+  id            String   @id @default(auto()) @map("_id") @db.ObjectId
```
Comment (vivekbarsagadey) [inline admin-ui/prisma/schema.prisma:45]: created and update is missing plz check

File: admin-ui/prisma/schema.prisma [hunk 5]
```diff
@@ -32,7 +32,7 @@ model Account {
 }
 
 model Session {
-  id           String   @id @default(cuid())
+  id           String   @id @default(auto()) @map("_id") @db.ObjectId
   sessionToken String   @unique @map("session_token")
```
Comment (vivekbarsagadey) [inline admin-ui/prisma/schema.prisma:36]: create and update is missing

File: admin-ui/pages/api/projects/[id].ts [hunk 6]
```diff
@@ -2,7 +2,7 @@ import type { NextApiRequest, NextApiResponse } from "next";
 import prisma from "../../../utils/prisma";
 import { handleApiMiddleware } from "../middleware";
 
-// DELETE /api/project/:id
+
```
Comment (vivekbarsagadey) [inline admin-ui/pages/api/projects/[id].ts:5]: put the ui comment like old one

File: admin-ui/prisma/schema.prisma [hunk 7]
```diff
@@ -116,34 +114,37 @@ model Project {
...
+  updatedAt     DateTime @updatedAt
+  name          String?
+  projectData   Json?
+
 
-model VerificationToken {
-  identifier String
-  token      String   @unique
-  expires    DateTime
+  @@map("tb_projectjson")
```
Comment (vivekbarsagadey) [inline admin-ui/prisma/schema.prisma:147]: it should be project_ json

Comment (SaurabhChandel007) [reply admin-ui/prisma/schema.prisma:147]: @@map("tb_project_json")

**Review by vivekbarsagadey — COMMENTED** [review]
.env.example is missing, plz add this

**Review by vivekbarsagadey — COMMENTED** [review]
VerificationToken is missing

**Pull request merged**